cli/compose/loader/loader.go

```diff
@@ -8,6 +8,7 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
This doesn't look correct; the repository was renamed from `Sirupsen/logrus` to `sirupsen/logrus` (see https://github.com/docker/cli/commit/a0113c3a448cb1ed875f2116eb8436ee8bd97bfc, and https://github.com/docker/cli/pull/424)

The correct import path is now with a lowercase `s`: https://github.com/sirupsen/logrus/issues/553